frontend/views/components/Forms/Password.vue

```diff
@@ -3,21 +3,21 @@
   i18n.label(tag='label' v-if='label') {{ label }}
   .input-combo
     input.input(
-      :type="isLock ? 'password' : 'text'"
+      :type='isLock ? "password" : "text"'
       :id='name'
-      :class="[{'error': v[name].$error}, size]"
+      :class='[{error: v[name].$error}, size]'
       :name='name'
-      :placeholder="showPlaceholder ? name : ''"
+      :placeholder='showPlaceholder ? name : ""'
       :data-test='name'
       v-model='value[name]'
       @input='v[name].$touch()'
```

Comment:
Here it is too!

Reply:
it is what?

Reply:
Existing. Instead of `$v`. I'm confused about what `v` is.

Reply:
I'll take a look at this later and let you know when I find the answer

Reply:
Thanks @sandrina-p! I opened an issue for this: #661